### PR TITLE
[docs]Update devconsole section in the docs

### PR DESCRIPTION
**Pull request by chetan-rns**
**What type of PR is this?**

> /kind cleanup

**What does this PR do / why we need it**:

* Remove the old setup guide since we already have Day 1 and Day 2 instructions
* Add images explaining the steps

**Have you updated the necessary documentation?**

* [x] Documentation update is required by this PR.
* [x] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: docs/devconsole/README.md [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+
+![Applications](../journey/day1/img/ArgoCD_UI.png)
+
+2. Install the GitOps Service operator from OperatorHub in all-namespaces
+
+![Operator](./img/GitOps_Operator.jpg)
+
+![Installation](./img/GitOps_Installation.jpg)
+
+3. The operator will install the GitOps backend service in `openshift-pipelines-app-delivery` namespace. `Application Stages` nav-item should now be visible (feature-flagged on the availability of the operator)
```
Comment (bigkevmcd) [inline docs/devconsole/README.md:16]: I'm not sure I'd say "feature-flagged" here, it's controlled by the availability of the operator.

**Review by bigkevmcd — APPROVED** [review]
Looks ok to me, I'd probably end the sentences with a `.` but otherwise, it's fine.

File: docs/devconsole/README.md (hunk 1 above)
Comment (chetan-rns) [reply docs/devconsole/README.md:16]: Resolved it here https://github.com/redhat-developer/kam/pull/60

**Pull request merged**